src/ui/dfn-list.js

```diff
@@ -41,7 +41,7 @@ ul.addEventListener("click", ev => {
 });
 
 function show() {
-  const definitionLinks = Object.entries(definitionMap)
+  const definitionLinks = Array.from(definitionMap.entries())
```

Comment:
```suggestion
  const definitionLinks = Array.from(definitionMap)
```

`.entries()` is redundant 👀